doc/configuration-flow.md

```diff
@@ -0,0 +1,54 @@
+# Overview of the configuration flow in Istio Manager
+
+Istio configuration consists of configuration objects. Each object has kind
+(for example, a routing rule), name and namespace uniquely identifying the
+object.  Istio Manager provides a REST-based interface for manipulating objects
+by their keys. Objects can be listed by their kind and/or namespace, retrieved
+by the object keys, deleted, created (using "POST" operation), and updated
+(using "PUT" operation).
+
+The configuration registry also provides a caching controller interface that
+also allows watching for changes to the configuration store. For example, a
+function can subscribe to receive object creation events for objects of a
+particular kind. The controller streams the events to the subscribers, and also
+maintains a local cache view of the entire store.
+
+The flow of configuration starts at the operator, continues to the
+platform-specific persistent store, and then triggers a notification event at
+the remote proxy component.
+
+## User input
```

Comment:
Can you number the sections so that its easier to map to the flow of configuration?